Runtime/JSValue.cs

```diff
@@ -249,6 +249,17 @@ public static unsafe JSValue CreateBigInt(int signBit, ReadOnlySpan<ulong> words
     public static implicit operator JSValue(ulong value) => CreateNumber(value);
     public static implicit operator JSValue(float value) => CreateNumber(value);
     public static implicit operator JSValue(double value) => CreateNumber(value);
+    public static implicit operator JSValue(bool? value) => value.HasValue ? GetBoolean(value.Value) : JSValue.Null;
```

Comment:
>JSValue.Null [](http://example.com/codeflow?start=103&length=12)

Somewhere else in code we map C# null concept to JSValue.Undefined. 
I believe the JSValue.Undefined is a much better match than JSValue.Null. The JSValue.Undefined is the default value for JSValue.
When we return napi_value from a function the Node-API considers C++/C# null to be the same as napi_undefined.

Reply:
I don't agree. If I call a C# method from JS, and the C# method returns `null`, I would expect to get a `null` value back in JS. Only `void` methods in C# (or property setters) should return `undefined`. Or maybe other situations where the value is truly uninitialized or unknown. I don't see anywhere else that's mapping C# `null` to JS `undefined`.

Reply:
We'll postpone this for now; I opened https://github.com/users/jasongin/projects/1/views/1?pane=issue&itemId=19832782 to track it for later.